pom.xml

```diff
@@ -638,6 +638,27 @@
           </execution>
         </executions>
       </plugin>
+      
+      <!--  Validate all XML and XSL in the project during the validate phase -->
+      <plugin>
+        <groupId>org.openjax.xml</groupId>
+        <artifactId>xml-maven-plugin</artifactId>
+        <version>0.9.2</version>
```

Comment:
Put the version in a variable (e.g. `xml.plugin.version`) and put the variable with the value up in the properties file of the POM file. Consult how the other plugins in the POM are done for an example.